src/hdmf/spec/spec.py

```diff
@@ -461,23 +468,26 @@ def id_key(cls):
     def type_key(cls):
         ''' Get the key used to store data type on an instance
 
-        Override this method to use a different name for 'data_type'
+        Override this method to use a different name for 'data_type'. HDMF supports combining schema
```

Comment:
I truly apologize for this decision

Reply:
All good. Hopefully this is the last of the buggy edge cases to address.